presto-tests/src/test/java/com/facebook/presto/tests/TestQueryManager.java

```diff
@@ -57,12 +58,20 @@
 public class TestQueryManager
 {
     private DistributedQueryRunner queryRunner;
+    private static final String LONG_LASTING_QUERY = "SELECT COUNT(*) FROM blackhole.default.dummy";
 
     @BeforeClass
     public void setUp()
             throws Exception
     {
-        queryRunner = TpchQueryRunnerBuilder.builder().build();
+        queryRunner = getSimpleQueryRunner();
+        queryRunner.installPlugin(new BlackHolePlugin());
+        queryRunner.createCatalog("blackhole", "blackhole");
+        // Black hole connectors do not have external metadata, which means that table metadata is local to each coordinator.
+        // So for the purposes of the test setup, create a local table per connector.
+        for (int i = 0; i < queryRunner.getCoordinators().size(); i++) {
```

Comment:
The referenced tests use multiple coordinators, however this one just uses a single coordinator, so we can remove the comment and the for loop and just use `queryRunner.getCoordinator()`.